resources/js/components/ui/chart.tsx

```
@@ -579,7 +589,7 @@ const ChartLegendContent = ({
             key={key}
             id={key}
             className={twMerge(
-              "flex items-center gap-2 rounded-sm px-2 py-1 text-muted-foreground *:data-[slot=icon]:-mx-0.5 *:data-[slot=icon]:size-2.5 *:data-[slot=icon]:shrink-0 *:data-[slot=icon]:text-muted-foreground",
+              "*:data-[slot=icon]:-mx-0.5 flex items-center gap-2 rounded-sm px-2 py-1 text-muted-foreground *:data-[slot=icon]:size-2.5 *:data-[slot=icon]:shrink-0 *:data-[slot=icon]:text-muted-foreground",
```

Biome (2.4.13)

[error] These CSS classes should be sorted.

(lint/nursery/useSortedClasses)